dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -201,7 +201,7 @@ public void VerifyCurrentAnchor()
                 // Test is failing in chk configuration due to:
                 // Bug #1726 Test Failure: RepeaterTests.VerifyCurrentAnchor 
                 Log.Warning("Skipping test for Debug builds.");
-                return;
+                //return;
```

Comment:
Uncomment?

Reply:
Oh missed that completely, thanks for pointing that out!